packages/waku/src/lib/renderers/rsc-renderer.ts

```diff
@@ -104,6 +104,22 @@ export async function renderRsc(
     },
   );
 
+  const serverBundlerConfig = new Proxy(
+    {},
+    {
+      get(_target, encodedId: string) {
+        const [fileId, name] = encodedId.split('#') as [string, string];
+        const id = filePathToFileURL(fileId);
+        if (fileId.startsWith('@id/assets/')) {
+          const id = '.' + fileId.slice('@id'.length);
+          return { id, chunks: [id], name, async: true };
```

Comment:
it will be `./assets/rsf0.js#func`

and because rsdw.js is in dist/rsdw.js, it import() will be work